### ISSUE TITLE
Stan model compilation is broken

### ISSUE THREAD

**Issue by mohamed82008** [TuringLang/TuringBenchmarks#8]
This is causing test failures. The following is a MWE that produces the error. cmdstan 2.18.0 is used.

```julia
## Optional to get cmdstan and save its url in TuringBenchmarks.CMDSTAN_HOME, can just define cmdstan_home instead if already got the program somewhere else
# using Pkg
# Pkg.build("TuringBenchmarks")

using Turing, TuringBenchmarks, Stan;
cmdstan_home = TuringBenchmarks.CMDSTAN_HOME
Stan.set_cmdstan_home!(cmdstan_home);
benchmark_models(["bernoulli-stan"], send=false)
```

The main failure is:

```
An error occurred while compiling the Stan program.

Please check your Stan program in variable 'bernoulli' and the contents of F:\tmp\bernoulli.exe_build.log.

ERROR: LoadError: Return code = -3
```

The log file is empty.

**Comment by yebai** [comment]
This is caused on an issue in setting stan path, which should be fixed by #8. Note `using MCMCChain` is also required to run the example.

**Comment by mohamed82008** [comment]
Ok thanks to your PR, I figured out some problems with the program. Now it works fine on Linux but gives the same error above on Windows. I will continue development on a Linux machine for now and try to figure out what's wrong with Windows support later. Thanks for the help.
